<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsePanel</class>
 <widget class="QWidget" name="UsePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#funcBar{
border-left: 1px solid #e0e0e0;
}
#groupFrame2{border-bottom:1px solid #e0e0e0;}
#imageFrame{
border: 1px solid #c8e1ff;
border-top:none;
background: #f1f8ff;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="groupFrame2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2" rowspan="6">
       <widget class="QLabel" name="patientIcon">
        <property name="pixmap">
         <pixmap resource="../ortrac.qrc">:/res/patient-128.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="Ui::FlatEdit" name="idEdit">
        <property name="placeholderText">
         <string>扫描或输入患者ID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="Ui::FlatEdit" name="deskEdit">
        <property name="placeholderText">
         <string>手术台号（可选）</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="Ui::FlatEdit" name="roomEdit">
        <property name="placeholderText">
         <string>手术室（可选）</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>242</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>247</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QFrame" name="funcBar">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="1" column="0">
       <widget class="QPushButton" name="submitButton">
        <property name="text">
         <string>确认请扫二维码或点此按钮</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="submitQr">
        <property name="pixmap">
         <pixmap resource="../ortrac.qrc">:/res/910108.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="pkgFrame">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>80</number>
      </property>
      <property name="topMargin">
       <number>40</number>
      </property>
      <property name="rightMargin">
       <number>80</number>
      </property>
      <property name="bottomMargin">
       <number>60</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="pixmap">
           <pixmap>:/res/uncheck-24.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>请扫描物品包上的条码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="bcInputButton">
          <property name="toolTip">
           <string>手工输入条码</string>
          </property>
          <property name="icon">
           <iconset resource="../ortrac.qrc">
            <normaloff>:/res/write-36.png</normaloff>:/res/write-36.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="noBcButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>无条码包登记</string>
          </property>
          <property name="icon">
           <iconset resource="../ortrac.qrc">
            <normaloff>:/res/minus-24.png</normaloff>:/res/minus-24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>229</width>
            <height>33</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="PackageView" name="pkgView"/>
      </item>
      <item>
       <widget class="QFrame" name="imageFrame">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>512</width>
             <height>256</height>
            </size>
           </property>
           <property name="pixmap">
            <pixmap resource="../ortrac.qrc">:/res/package.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ui::FlatEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">ui/inputfields.h</header>
  </customwidget>
  <customwidget>
   <class>PackageView</class>
   <extends>QTableView</extends>
   <header>controls/packageview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ortrac.qrc"/>
 </resources>
 <connections/>
</ui>
